ansible/roles/ci/code_style_check/license/files/gather_missing_licences.py

```diff
@@ -78,10 +79,24 @@ def get_changes_in_pr(data):
         print(f"ERROR WITH COMMAND:\nstderr:{master_branch_process.stderr}\nstdout:{master_branch_process.stdout}")
         exit(1)
     devel_branches = ""
-    for branch in master_branch_process.stdout.split("\n"):
-        if any(entry in branch for entry in MASTER_BRANCHES):
-            devel_branches = branch.strip()
-            break
+    all_branches = master_branch_process.stdout.split("\n")
+    all_branches = [branch.strip() for branch in all_branches]
```

Comment:
list_of_accepted_master_branches = [...]
all_branches = [branch for branch in all_branches if branch.strip() in list_of_accepted_master_branches]